scipy/stats/_continuous_distns.py

```diff
@@ -2692,6 +2692,9 @@ def _stats(self, c):
         g2 /= mu2**2.0
         return mu, mu2, g1, g2
 
+    def _entropy(self, c):
+        return sc.betaln(c, 1) + 1 + sc.psi(c) + 1/c - sc.psi(1)
```

Comment:
I noted in my earlier comment that the beta function simplifies for the case of `b=1` (and that's why SciPy is a special case of the type 4 generalized logistic)
```suggestion
        return -np.log(c) + 1 + sc.psi(c) + 1/c - sc.psi(1)
```